examples/cisco/demo_vrf.pp

```diff
@@ -16,9 +16,19 @@
 
 class ciscopuppet::cisco::demo_vrf {
 
+  $rt_export_stitching = $operatingsystem ? {
+    'ios_xr' => ['22:13 stitching', '24:15 stitching'],
```

Comment:
Basic question: Why does the manifest have to specify `stitching` for this property? I mean, the property itself defines this as a stitching command so what is gained by specifying it in the manifest? It doesn't seem to be an optional keyword.